test/manual/html/perf.html

```diff
@@ -0,0 +1,36 @@
+<!doctype html>
+<html lang="en" class="has-background-light">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0, user-scalable=no, target-densitydpi=medium-dpi">
+    <title>Flicking - QA</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/bulma/0.7.4/css/bulma.min.css">
+    <link rel="stylesheet" href="../css/common.css">
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/es5-shim/4.1.5/es5-shim.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/es5-shim/4.1.5/es5-sham.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/fpsmeter/0.3.1/fpsmeter.min.js"></script>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/Swiper/4.5.0/css/swiper.min.css">
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/Swiper/4.5.0/js/swiper.js"></script>
+    <script src="../../../dist/flicking.pkgd.js"></script>
+</head>
+<body class="container" style="display: none;">
+    <section class="hero">
+        <div class="hero-body">
+            <h1 class="title">
+                Performance Test
+            </h1>
+        </div>
+    </section>
+
+    <p class="subtitle is-4">Original</p>
```

Comment:
How about `Flicking` instead of `Original`?

Reply:
That word is totally meaningless. That's just leftover while I'm comparing new methods(display: none, removing element) to the original one.
I think removing that rather be nice.